lib/client.js

```diff
@@ -322,7 +322,9 @@ exports.createClient = function(reference, name) {
             if (reportStats) {
               reportStats(true, 'http-' + res.status);
             }
-            if(res.headers['content-type'].indexOf('application/json') === -1 || !res.body){
+            if((_.includes(res.headers, 'content-type') &&
```

Comment:
hmm... maybe I'm reading the docs wrong... Or are we missing some tests, because ideally this should be caught by travis...